site/content/posts/2021-06-10-docs-wg.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Contour Technical Documentation Working Group Kick Off
+excerpt: Announcing Contour Technical Documentation Working Group
+author_name: Contour Team
+author_avatar: /img/contributors/orlin-vasilev.jpg
+categories: [kubernetes]
+tags: ['Contour Team', 'Abigail McCarthy', 'Docs' ,'Docs-WG', 'Jonas Rosland', 'Orlin Vasilev']
+date: 2021-06-10
+slug: docs-wg
+---
+
+
+
+# Join the Contour Tech Docs Working Group
+
+The Contour team is excited to announce we are launching a technical documentation working group. This group will focus on improving the Contour website, documentation, and processes around content creation.
+
+# Because documentation makes a difference
+
+In 2017, GitHub conducted the Open Source Survey, which sampled 5,500 respondents. One of the key takeaways was that “documentation is highly valued, but often overlooked.” The survey found that outdated or confusing documentation was a major problem, affecting 93% of respondents. Yet 60% of contributors rarely or never contribute to documentation. We want this working group to be an opportunity to give back to the community by helping to improve the docs.
+
+# The purpose and scope of the Working Group
+
+The purpose of the Working Group is to build up a self-sustaining community around documentation for Contour. We have an initial need to rework the existing documentation based on the recommendations outlined in the CNCF Tech Docs Review, and a continuous need for documentation being added/edited/removed for releases going forward.
+This group is open to contributors of all levels, the only requirement is being interested in helping with docs!
+
+# HOW TO PARTICIPATE
+Join us for our kickoff meeting on June 17th at 1PM Eastern Standard/Daylight Time as part of the Contour Office Hours. During the kickoff meeting, we’ll have an information session about this working group, covering the following topics
```

Comment:
Missing link to time zone checker: https://www.timeanddate.com/worldclock/usa/boston